app/models/comments.rb

```diff
@@ -0,0 +1,11 @@
+class Comments < ApplicationRecord
+  belongs_to :users, foreign_key: 'author_id', class_name: 'User'
```

Comment:
- [ ] Please remember that `belongs_to` should use a singular term. If you use the pluralized form in the above example for the `user` association in the Comment model and try to create the instance by `Comment. create(users: @user),` you would be told that there was an "uninitialized constant Comment::Users".
so it should be `belongs_to :user`